nbconvert/preprocessors/svg2pdf.py

```diff
@@ -20,6 +20,12 @@
 
 INKSCAPE_APP = '/Applications/Inkscape.app/Contents/Resources/bin/inkscape'
 
+if sys.platform == "win32":
+    if sys.version_info < (2, 99):
```

Comment:
Probably best to use `try import/except import` instead of targeting a specific version